charts/templates/service.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   namespace: {{ .Values.namespace}}
   labels:
     {{- include "marklogic.labels" . | nindent 4 }}
+  annotations:
+{{- range $key, $value := .Values.service.annotations }}
```

Comment:
@rwinieski This functionally work well, however, how about using the code below so that it is consistent with other places that using values file, and a bit more concise?
    {{- toYaml .Values.service.annotations | nindent 4 }}

Reply:
@pengzhouml I just committed this change.